egs2/dirha_wsj/asr1/path.sh

```diff
@@ -0,0 +1,31 @@
+MAIN_ROOT=$PWD/../../..
+KALDI_ROOT=$MAIN_ROOT/tools/kaldi
+
+[ -f $KALDI_ROOT/tools/env.sh ] && . $KALDI_ROOT/tools/env.sh
```

Comment:
You don't need to copy this file. Would you change to symbolic link?

Reply:
Done.